### PR TITLE
[frontend] add button to navigate back to scenario from simulation (#1611)

### LINKED ISSUE #1611: Be able to access to the original scenario from simulation (going to parent)
## Use case

Be able to access to the original scenario from simulation (going to parent)

### PR DESCRIPTION
**Pull request by antoinemzs**
### Proposed changes

* Adding a way to navigate back to the parent scenario from a simulation
*

### Related issues

* Closes #1611
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments
image
image
image
image

### CODE REVIEW

File: openbas-front/src/admin/components/simulations/simulation/Index.tsx [hunk 1]
```diff
@@ -173,6 +174,9 @@ const Index = () => {
       setLoading(false);
     });
   });
+  useDataLoader(() => {
+    dispatch(fetchScenario(exercise.exercise_scenario));
```
Comment (savacano28) [inline openbas-front/src/admin/components/simulations/simulation/Index.tsx:178]: Do we really need to fetch the entire scenario just to display a name? Why not have a button that says 'Go to Parent Scenario' instead?

File: openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx [hunk 2]
```diff
@@ -18,11 +23,18 @@ interface Props {
 const ExerciseMainInformation: React.FC<Props> = ({ exercise }) => {
   const { t } = useFormatter();
   const sortByOrder = R.sortWith([R.ascend(R.prop('phase_order'))]);
+  const scenarioBaseUri = '/admin/scenarios';
+  const scenarioId: string = exercise.exercise_scenario || '';
+  const { scenario } = scenarioId
```
Comment (savacano28) [inline openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx:28]: If I remove the parent scenario, I get this error on exercise side:
image
image

**Review by savacano28 — CHANGES_REQUESTED** [review]

File: openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx (hunk 2 above)
Comment (antoinemzs) [reply openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx:28]: 💀

File: openbas-front/src/admin/components/simulations/simulation/Index.tsx (hunk 1 above)
Comment (antoinemzs) [reply openbas-front/src/admin/components/simulations/simulation/Index.tsx:178]: I've checked with product re: UX and design and the outcome was that it was an ok spot with displaying the scenario name. Fetching a single scenario is probably fine in this instance (i.e. it's not a collection).

File: openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx (hunk 2 above)
Comment (antoinemzs) [reply openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx:28]: Ok I had made a mistake in null checking, I fixed it in the latest commit

**Comment by savacano28** [comment]
Tested OK

**Pull request merged**